src/compass/s1_rdr2geo.py

```diff
@@ -116,7 +116,12 @@ def run(cfg, burst=None, save_in_scratch=False):
                        rdr2geo_cfg.compute_local_incidence_angle,
                        gdal.GDT_Float32),
                        'heading': (
-                       rdr2geo_cfg.compute_azimuth_angle, gdal.GDT_Float32)}
+                       rdr2geo_cfg.compute_azimuth_angle, gdal.GDT_Float32),
+                       'los_east': (
+                       rdr2geo_cfg.compute_ground_to_sat_east, gdal.GDT_Float32),
+                       'los_north': (
```

Comment:
`los_east` and `los_north` (as well other topo layer file names) are hard coded here and in `s1_geoocde_metadata.py`. What do you think about consolidating the files names into one place that can be imported from `rdr2geo.py`? e.g. Do the following to `s1_rdr2geo.py`:
```python
# list of file names for rasters created by rdr2geo
...
file_name_los_east = 'los_east'
file_name_los_north = 'los_north'

def run():
...
    topo_output = {
    file_name_los_east: (rdr2geo_cfg.compute_ground_to_sat_east, gdal.GDT_Float32),
...
}
```
Then in `s1_geocode_metadata.py` do the following:
```python
from rdr2geo import file_name_los_east, ...
...
def run():
    ...
    static_layers = {
        ...
        'east_los_vector': (cfg.rdr2geo_params.compute_ground_to_sat_east,
                            file_name_los_east),
        ...
    }
```